src/config.js

```diff
@@ -138,4 +140,19 @@ export default {
     '42': 'api-kovan',
     '4': 'api-rinkeby',
   },
+
+  /**
+   * latest block sync timeout
```

Comment:
Not sure if we should write comments like this. Variable name is already explaining its purpose.